### DISCUSSION TITLE
Bull Dashboard

### DISCUSSION THREAD

**Discussion by kasgithub8** [Budibase/budibase#9560] [Help]
Hi,

I wonder if anybody could help me understand the bull dashboard for our automations.

I really like the idea behind it and would want to utilize it but the website is a bit wild.

Can somebody help me with this or are there settings for it?

Thanks!

**Comment by melohagan** [comment]
Hey @kasgithub8 

We only use Bull Dashboards for debugging purposes. We don't expose or support it.
